conf/local.conf.sample

```diff
@@ -280,3 +280,14 @@ BB_DISKMON_DIRS = "\
 # track the version of this file when it was generated. This can safely be ignored if
 # this doesn't mean anything to you.
 CONF_VERSION = "1"
+
+# WIC image type support
+IMAGE_FSTYPES ?= "tar.bz2 wic.bz2"
+
+# dosfstools is needed to create the boot partition, mtools to copy to it
+IMAGE_DEPENDS_wic_append = " dosfstools-native mtools-native"
+
+# Uncomment one of these for specific SD card sizes (default=4g)
+WKS_FILE ?= "sdimage-bootpart-4g.wks"
+# WKS_FILE ?= "sdimage-bootpart-2g.wks"
+# WKS_FILE ?= "sdimage-bootpart-8g.wks"
```

Comment:
For the machines that require machines-specific changes in wks files, we will use a set of wks files with the same name in a more specific layer but you're right, it won't work with qemu machines.